extensions/wikia/GlobalNavigation/scripts/GlobalNavigationSearch.js

```diff
@@ -40,6 +41,14 @@ require(['jquery', 'wikia.browserDetect', 'GlobalNavigationiOSScrollFix'], funct
 	$(function () {
 		var $globalNav = $('#globalNavigation');
 
+		//Checks whether the searchInput is empty and prevents any action if true
+		$searchForm.submit(function (e) {
+			if (!$searchInput.val()) {
+				e.preventDefault();
+				e.stopPropagation();
```

Comment:
you can do "return false" instead of calling preventDefault() and stopPropagation()